### PR TITLE
Implemented #515.

### PR DESCRIPTION
**Pull request by LinoxGH**
Implemented the nagging feature from API to MiniMessage. But because API wasn't a part of MiniMessage I put down a small method and used that.

### CODE REVIEW

File: text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/MiniMessageParser.java [hunk 1]
```diff
@@ -247,4 +250,10 @@ private Component handleModifying(final Modifying modTransformation, final Compo
   private String sanitizePlaceholderName(final String name) {
     return name.toLowerCase(Locale.ROOT);
   }
+
+  private void tryToNag(final String input, final boolean strict) {
```
Comment (zml2008) [inline text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/MiniMessageParser.java:254]: Maybe just call this in `TokenParser`? I believe that would capture all code paths at once.

**Review by zml2008 — CHANGES_REQUESTED** [review]

File: text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/MiniMessageParser.java (hunk 1 above)
Comment (kezz) [reply text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/MiniMessageParser.java:254]: Yeah, perhaps doing it as late as possible is the better solution?

File: text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/parser/TokenParser.java [hunk 2]
```diff
@@ -503,6 +503,8 @@ private static ElementNode buildTree(
       throw new ParsingException(sb.toString(), message, errorTokens);
     }
 
+    tryToNag(message, strict);
```
Comment (zml2008) [inline text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/parser/TokenParser.java:506]: Why are you running this so late? Plus, buildTree isn't used for the strip and escape tokens methods.

**Review by zml2008 — CHANGES_REQUESTED** [review]

**Review by zml2008 — APPROVED** [review]

**Comment by zml2008** [comment]
I've rebased this and tweaked the implementation to be fully within the parser -- would appreciate another set of eyes over it though.

**Pull request merged**